zulip_bots/zulip_bots/run.py

```diff
@@ -73,11 +80,13 @@ def parse_args():
                         help='Install dependencies for the bot.')
     options = parser.parse_args()
 
-    if not options.name and not options.path_to_bot:
+    if (not options.name and not options.path_to_bot) or is_not_equal(options.name, options.path_to_bot):
```

Comment:
This logic fails for me when I try to only specify a bot name: In such a case, `options.name` is not equal to `options.path_to_bot` (which is probably None).